proof/invariant-abstract/AInvs.thy

```diff
@@ -1886,6 +1887,10 @@ lemma check_budget_restart_schact_is_rct_imp_ct_activatable[wp]:
   "check_budget_restart \<lbrace>\<lambda>s. schact_is_rct s \<longrightarrow> ct_in_state activatable s\<rbrace>"
   by (wpsimp simp: check_budget_restart_def)
 
+lemma update_reply_schact_is_rct_imp_ct_activatable[wp]:
```

Comment:
`update_reply` won't touch the scheduler action or the TCB heap, so with the right rules, this should be `crunch`able, and we won't need to unfold low-level things like `set_object` and `get_object`.